src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ValidationService.cs

```diff
@@ -55,7 +72,8 @@ private static (bool Valid, IEnumerable<ReleaseStatusLogMessage> LogMessages) Va
         {
             return release.PublishScheduled.HasValue
                 ? Success()
-                : Failure(ValidationStage.ReleaseMustHavePublishScheduledDate, $"Scheduled publish date status is not set");
+                : Failure(ValidationStage.ReleaseMustHavePublishScheduledDate,
+                    $"Scheduled publish date status is not set");
```

Comment:
Is this correct - the status of a date? That would confuse me if I got that message.

Reply:
I've corrected the message now, thanks